pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -66,6 +66,7 @@ type CloudProvider struct {
 }
 
 func NewCloudProvider(ctx context.Context, options cloudprovider.Options) *CloudProvider {
+	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).Named("aws"))
```

Comment:
This is just a bunch, but do the logs indent more cleanly if if name this `cloudprovider.aws` since the width is more similar to `provisioning` `selection`

Reply:
I'd rather keep it AWS since it's just startup logic so the logging is actually nice. When it actually matters in the async process the indentation is nice since it's `aws.launchtemplate` which lines up with `provisioning` etc